datumaro/plugins/yolo_format/extractor.py

```diff
@@ -287,11 +315,321 @@ def get_subset(self, name):
         return self._subsets[name]
 
 
-class YoloImporter(Importer):
-    @classmethod
-    def detect(cls, context: FormatDetectionContext) -> None:
-        context.require_file("obj.data")
+class Yolo8Extractor(YoloExtractor):
+    RESERVED_CONFIG_KEYS = Yolo8Path.RESERVED_CONFIG_KEYS
+
+    def __init__(
+        self,
+        *args,
+        config_file=None,
+        **kwargs,
+    ) -> None:
+        super().__init__(*args, **kwargs)
```

Comment:
@Eldies , do we have a test on custom config name for the extractor?

Reply:
yes, here: https://github.com/cvat-ai/datumaro/pull/50/files#diff-f9d553feac7bf5be22aab5dcfb4db1a75aa5806e87860e09a48b2c2c4ff1ae7dR820